vvv-config.yml

```diff
@@ -11,6 +11,15 @@ sites:
       - src.wordpress-develop.dev
       - build.wordpress-develop.dev
 
+  # This would create a standard WordPress site in www/example-site available
+  # at http;//my-example-site.dev
+  # Remember to copy this file to vvv-custom.yml before making changes
+
+  #example-site:
+  #  repo: https://github.com/Varying-Vagrant-Vagrants/vvv-wordpress-default.git
+  #  hosts:
+  #    - my-example-site.dev
```

Comment:
do we have a generic WP install that can replace it? Similar to https://github.com/tomjn/vvv-wp-multisite-subdir ?